packages/manager/apps/zimbra/src/pages/dashboard/Organizations/Organizations.tsx

```diff
@@ -97,8 +61,52 @@ export default function Organizations() {
     console.log('Ajouter une nouvelle organisation');
   };
 
+  const columns: DatagridColumn<OrganizationItem>[] = [
+    {
+      id: 'name',
+      cell: (item) => <IdLink id={item.id}>{item.name}</IdLink>,
+      label: 'zimbra_organization_name',
```

Comment:
Do the translation directly in the definition instead of doing it in the map function
```
columns={columns.map((column) => ({
          ...column,
label: t(column.label),
```
```suggestion
      label: t('zimbra_organization_name'),
```